docs/en_US/Release.rst

```diff
@@ -5,6 +5,69 @@
 Change Log
 ==========
 
+Release 2.1 - 3/8/2021
+-----------------------
+
+Major updates
+^^^^^^^^^^^^^
+
+Neural architecture search
+""""""""""""""""""""""""""
+
+* Improve NAS 2.0 (Retiarii) Framework (Alpha release)
+
+  * `Feature roadmap <https://github.com/microsoft/nni/issues/3301>`__
+  * `Related issues and pull requests <https://github.com/microsoft/nni/issues?q=label%3Aretiarii-v2.1>`__
+  * Documentation improvement (#3404 #3432)
+
+Training service
+""""""""""""""""
+
+* Support shared storage for reuse mode (#3354)
```

Comment:
do we have official name for "reuse mode"? "reuse mode" seems too adhoc...

Reply:
The document is like [this](https://nni.readthedocs.io/en/latest/reference/experiment_config.html?#reusemode), maybe a new name is needed?

Reply:
it is fine for now